LabExT/View/Movement/CoordinatePairingsWindow.py

```diff
@@ -161,6 +172,24 @@ def __setup__(self) -> None:
                 frame, self.mover.output_calibration).pack(
                 side=TOP, fill=X)
 
+        if self.experiment_manager:
+            shortcuts_frame = CustomFrame(self._main_frame)
+            shortcuts_frame.pack(side=TOP, fill=X, pady=5)
+
+            search_for_peak_button = Button(
+                shortcuts_frame,
+                text="Perform Search for Peak...",
+                command=self._on_perform_search_for_peak)
```

Comment:
Suggestion: I would use the function `self.experiment_manager.main_window.open_live_viewer` here, so you can re-use all the top-level checking logic from the main window.